### PR TITLE
Updating master to 4.1.0

### PR DESCRIPTION
**Pull request by InsertCreativityHere**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,42 @@ This file describes the change history for the Ice Builder for Eclipse
 plug-in.
 
 ======================================================================
+Changes since version 4.0.0.20150721
```
Comment (spruiell) [inline CHANGELOG.md]: The changelog should be cleaned up to use real markdown

File: README.md [hunk 2]
```diff
@@ -60,50 +44,65 @@ Alternately, you can install the plug-in from the Eclipse Marketplace.
...
+Upon activation, the project immediately creates a `generated` directory to hold the Java source files the Slice compiler generates from your Slice files.
+![Activating the Plug-in](/Screenshots/activation.png)
+To de-activate the plugin, simply navigate to the `Ice Builder` menu and instead select `Remove Ice Builder`. Note that removing the Ice Builder plug-in
+will not affect your Slice files, but will remove all generated code.
 
 ### Configuring Project Settings
 
-To configure the project-specific settings, select `Properties` from the `Project` menu or right-click on the name of your project and choose `Properties`. Click on `Ice Builder` to view the plug-in's configuration settings, which are presented in two tabs: Source and Options.
+To configure the project-specific settings, select `Properties` from the `Project` menu or right-click on the name of your project and choose `Properties`.
+Click on the `Ice Builder` menu to to view the plug-in's settings for the selected project.
```
Comment (spruiell) [inline README.md]: "to to"

File: README.md [hunk 3]
```diff
@@ -60,50 +44,65 @@ Alternately, you can install the plug-in from the Eclipse Marketplace.
...
+directory for this. To change the directory, you must first create the new directory, and then click `Browse` to navigate to it. The new directory must be empty,
+and the plug-in requires exclusive use of it; any files or changes made to the directory will be lost when the Slice compiler is run.
 
-#### Settings in the Source Tab
+Any external Slice files your project is dependent upon can be specified using the `Include Directories` interface, where you can add directories 
+the compiler can search for referenced Slice definitions. These directories are searched in the order they're listed in the interface.
 
-This tab configures the directories of your Slice files and generated code. The plug-in includes the `slice` subdirectory by default, but you can remove this directory and add other directories if necessary. The plug-in only compiles Slice files that are located in the configured subdirectories.
+For more advanced users, additional options can be passed directly to the Slice compiler with the `Advanced Options` section. For a list of all supported
```
Comment (spruiell) [inline README.md]: The project properties dialog currently calls this "Additional Options", not "Advanced Options"

File: README.md [hunk 4]
```diff
@@ -6,37 +6,21 @@ An [Ice](https://github.com/zeroc-ice/ice) installation with `slice2java` versio
...
-for PDE, then click install to install the Eclipse PDE.
-
-To import the project choose `File > Import > General > Existing Projects into Workspace`
-and select the java directory in this repository as the root directory.
-
-To create the plug-in, use `File > Export > Plug-in Development > Deployable plug-ins and fragments`.
+  - [Using the Plug-in](#using-the-plugin)
+- [Building the Plug-in from Source](#building-the-plugin-from-source)
 
 ## Usage
```
Comment (spruiell) [inline README.md:26]: Should we mention anything about obtaining the Ice JAR files, or is that an exercise for the reader?

Comment (InsertCreativityHere) [reply README.md:26]: Leave it as an exercise I'd say. Seems outside of the scope of the Builder to me.

**Pull request merged**